website/source/docs/enterprise/index.html.md

```diff
@@ -3,18 +3,20 @@ layout: "docs"
 page_title: "Nomad Enterprise"
 sidebar_current: "docs-enterprise"
 description: |-
-  Nomad Enterprise features a number of capabilities beyond the open source
-  offering that may be beneficial in certain workflows.
+  Nomad Enterprise adds collaboration and governance capabilities to Nomad.
+  Features include Namespaces, Resource Quotas and Sentinel Policies.
```

Comment:
What is our stance on the Oxford Comma? I am usually for it. Consul team also enforced it in our docs. I would vote to keep things consistent.

Reply:
That works for me. I added the one after "Resource Quotas" above. Let me know if have missed any others.